src/wrapped/wrappedpulse.c

```diff
@@ -869,11 +869,11 @@ static void* find_time_new_Fct(void* fct)
     return NULL;
 }
 // time_restart
-#define GO(A)                                               \
-static uintptr_t my_time_restart_fct_##A = 0;               \
-static void* my_time_restart_##A(void* e, void* t)          \
-{                                                           \
-    RunFunctionFmt(my_time_restart_fct_##A, "pp", e, t);    \
+#define GO(A)                                                          \
+static uintptr_t my_time_restart_fct_##A = 0;                          \
+static void* my_time_restart_##A(void* e, void* t)                     \
```

Comment:
Yeah, looks like it's `void` callback: https://freedesktop.org/software/pulseaudio/doxygen/structpa__mainloop__api.html

Reply:
Fixed.